docs/content/preview/tools/tableplus.md

```diff
@@ -34,7 +34,7 @@ To create a connection, do the following:
 
 1. In TablePlus, from the **Connection** menu, choose **New**.
 
-1. Select **PostgreSQL** for YSQL, or **Cassandra** for YCQL, and click **Create**.
+1. Select **PostgreSQL** for YSQL, or **Cassandra** for YCQL, and click **Create**. Note that JSONB data type is unsupported for YCQL.
```

Comment:
I would add a sentence in Before you begin section instead

Note that if you are connecting to a YCQL keyspace that has tables with columns of type JSONB, because the Cassandra driver used by TablePlus does not support JSONB, the connection to the database may fail.